client/context/helpers.go

```diff
@@ -345,3 +348,28 @@ func (ctx CoreContext) GetNode() (rpcclient.Client, error) {
 	}
 	return ctx.Client, nil
 }
+
+func getReadableString(a interface{}) {
```

Comment:
This function name isn't exactly indicative of what it's performing.

Reply:
OK. Could you give me some advice?